bundles/org.openhab.binding.qbus/src/main/java/org/openhab/binding/qbus/internal/handler/QbusCO2Handler.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.qbus.internal.handler;
+
+import static org.openhab.binding.qbus.internal.QbusBindingConstants.CHANNEL_CO2;
+import static org.openhab.core.types.RefreshType.REFRESH;
+
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.qbus.internal.QbusBridgeHandler;
+import org.openhab.binding.qbus.internal.protocol.QbusCO2;
+import org.openhab.binding.qbus.internal.protocol.QbusCommunication;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.types.Command;
+
+/**
+ * The {@link QbusCO2Handler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Koen Schockaert - Initial Contribution
+ */
+
+@NonNullByDefault
+public class QbusCO2Handler extends QbusGlobalHandler {
+
+    public QbusCO2Handler(Thing thing) {
+        super(thing);
+    }
+
+    protected @NonNullByDefault({}) QbusThingsConfig config;
+
+    int co2Id = 0;
+
+    @Nullable
+    String sn;
```

Comment:
Can these be private?
```suggestion
    private int co2Id = 0;

    private @Nullable String sn;
```